src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -14,16 +14,15 @@
 public class JdbcUserDao implements UserDao {
 
     private final String COUNT_ALL = "SELECT COUNT(*) FROM public.users";
-    private final String FIND_BY_EMAIL = "SELECT * FROM public.users" +
+    private final String FIND_BY_EMAIL = "SELECT email, password, salt, iterations, role" +
+            " FROM public.users" +
             " WHERE email=?";
     private final String INSERT = "INSERT INTO public.users" +
             " (email, password, salt, iterations, role)" +
             " VALUES (?, ?, ?, ?, ?)";
 
```

Comment:
if we write variable in UpperCase we should put on 'static' after private or if we don't write 'static' then we should write variable in LowerCase

Reply:
> if we write variable in UpperCase we should put on 'static' after private or if we don't write 'static' then we should write variable in LowerCase
We use UpperCase for naming constants. 
Are you sure that:
     private final String FIND_BY_EMAIL = "SELECT email, password, salt, iterations, role" +
            " FROM public.users" +
            " WHERE email=?";
is not a constant?

Reply:
> if we write variable in UpperCase we should put on 'static' after private or if we don't write 'static' then we should write variable in LowerCase

We use UpperCase for naming constants. 
Are you sure that:
     private final String FIND_BY_EMAIL = "SELECT email, password, salt, iterations, role" +
            " FROM public.users" +
            " WHERE email=?";
is not a constant?

https://www.tutorialspoint.com/what-is-a-constant-and-how-to-define-constants-in-java